### PR TITLE
ClassDetail: Treat null as NaN

### LINKED ISSUE #379: Points statistics not correct
I thing, point stats is not displaying correct data, see screenshot below:

points_stats

### PR DESCRIPTION
**Pull request by DanielKrasny**
Closes #379.

### CODE REVIEW

File: frontend/src/ClassDetail.svelte [hunk 1]
```diff
@@ -60,7 +60,7 @@
 
       for (let i = 0; i < clazz.students.length; i++) {
           const student = clazz.students[i];
-          if (!isNaN(clazz.assignments[assignment_index].students[student.username].assigned_points)) {
+          if (!isNaN(parseFloat(clazz.assignments[assignment_index].students[student.username].assigned_points))) {
```
Comment (Kobzol) [inline frontend/src/ClassDetail.svelte:63]: This is a bit hacky. Could we do something like this instead?
```suggestion
          const points = clazz.assignments[assignment_index].students[student.username].assigned_points;
          if (!isNaN(points) && points !== null) {
```

**Review by Kobzol — COMMENTED** [review]
Thanks for the quick fix :)

File: frontend/src/ClassDetail.svelte (hunk 1 above)
Comment (DanielKrasny) [reply frontend/src/ClassDetail.svelte:63]: Of course we could :) I was just afraid of creating a variable for that.

**Review by Kobzol — APPROVED** [review]
Thank you! :heart:

**Pull request merged**